src/ServiceControl.Audit/Auditing/AuditIngestionComponent.cs

```diff
@@ -35,11 +39,12 @@ class AuditIngestionComponent
             Metrics metrics,
             Settings settings,
             IDocumentStore documentStore,
+            IBodyStorage bodyStorage,
             RawEndpointFactory rawEndpointFactory,
             LoggingSettings loggingSettings,
-            BodyStorageFeature.BodyStorageEnricher bodyStorageEnricher,
-            IEnrichImportedAuditMessages[] enrichers,
-            AuditIngestionCustomCheck.State ingestionState
+            AuditIngestionCustomCheck.State ingestionState,
+            EndpointInstanceMonitoring endpointInstanceMonitoring,
+            IEnumerable<IEnrichImportedAuditMessages> auditEnrichers // allows extending message enrichers with custom enrichers registered in the DI container
```

Comment:
I had to bring this back in since this "feature" is being used by the acceptance tests. Not excited to keep this in just for the acceptance tests but than can be solved separately.